model/src/main/java/org/projectnessie/api/params/ReferencesParams.java

```diff
@@ -29,12 +32,29 @@
 
   public ReferencesParams() {}
 
-  private ReferencesParams(Integer maxRecords, String pageToken) {
+  @Parameter(
+      description =
+          "If set to true, will fetch additional metadata for branches, such as number of commits ahead/behind or the common ancestor in relation to the default branch, and the commit metadata for the HEAD commit.\n\n"
+              + "A returned Branch instance will then have a map of 'metadataProperties' with the following keys:\n\n"
+              + "- numCommitsAhead (number of commits ahead of the default branch)\n\n"
+              + "- numCommitsBehind (number of commits behind the default branch)\n\n"
+              + "- headCommitMeta (the commit metadata of the HEAD commit)\n\n"
+              + "- commonAncestorHash (the hash of the common ancestor in relation to the default branch).\n\n"
+              + "Note that computing & fetching additional metadata might be computationally expensive on the server-side, so this flag should be used with care.")
+  @QueryParam("fetchAdditionalInfo")
+  private boolean fetchAdditionalInfo;
+
+  public boolean isFetchAdditionalInfo() {
```

Comment:
nit: odd method order: getter above constructor?

Reply:
I'll fix it in the next PR that is coming